pkg/kubelet/kubelet.go

```diff
@@ -266,6 +268,7 @@ func NewMainKubelet(
 		nodeName:                       nodeName,
 		dockerClient:                   dockerClient,
 		kubeClient:                     kubeClient,
+		clientset:                      clientset,
```

Comment:
I make kubelet containing both the old client and the new clientset. I will remove the old client after when refactoring kubelet.